Fix workbooks deployment permissions documentation (#1629)
microsoft/finops-toolkit: docs/workbooks.md

## Deploy FinOps workbooks

> FinOps workbooks require the <strong>Contributor</strong> role or a role with both <strong>Microsoft.Resources/deployments/validate/action</strong> and <strong>Microsoft.Resources/deployments/write</strong> permissions for ARM template deployments, <strong>Workbook Contributor</strong> role to save changes, and <strong>Reader</strong> on all subscriptions you want to monitor.
> 
> > If you only have Reader access, you can download the workbook files from the [GitHub repository](https://github.com/microsoft/finops-toolkit/tree/main/src/workbooks) and import them directly into Azure Monitor Workbooks.

Reviewer: ```suggestion
FinOps workbooks require the <strong>Workbook Contributor</strong> role to deploy and <strong>Reader</strong> on any subscriptions you want to monitor.
```

> > If you only have Reader access, you can download the workbook files from the [GitHub repository](https://github.com/microsoft/finops-toolkit/tree/main/src/workbooks) and import them directly into Azure Monitor Workbooks.

Reviewer 2: ```suggestion
> If you only have Reader access, you can download the workbook JSON files from the finops-workbooks.zip package available in the [latest release](https://aka.ms/ftk/latest), and then import them directly into Azure Monitor Workbooks.
```
